paddle/fluid/framework/device_worker.h

```diff
@@ -207,54 +207,104 @@ class DownpourWorker : public HogwildWorker {
   void CopySparseTable();
   void CopyDenseTable();
   void CopyDenseVars();
-
- private:
-  bool need_dump_param_;
-  std::vector<std::string> dump_param_;
+  std::string PrintLodTensor(LoDTensor* tensor, int64_t start, int64_t end);
+  std::pair<int64_t, int64_t> GetTensorBound(LoDTensor* tensor, int index);
+  bool CheckValidOutput(LoDTensor* tensor, size_t batch_size);
```

Comment:
I think some variables can be declared in DownpourWorkerOpt.
btw, does the name of DownpourWorkerOpt has alternatives?

Reply:
All these variables are moved from private to protected. It should be clear without comments. working on alternatives